boards/nordic/nrf54h20dk/board.c

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+
+#define MODULE mram_suspend_off
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(MODULE);
+
+#include <services/nrfs_mram.h>
+#include <nrfs_backend_ipc_service.h>
+
+#define MRAM_SUSPEND_OFF_INIT_PRIO 90
+
+void mram_latency_handler(nrfs_mram_latency_evt_t const *p_evt, void *context)
+{
+	switch (p_evt->type) {
+	case NRFS_MRAM_LATENCY_REQ_APPLIED:
+		LOG_INF("MRAM latency handler: response received");
+		break;
+	case NRFS_MRAM_LATENCY_REQ_REJECTED:
+		LOG_ERR("MRAM latency handler - request rejected!");
```

Comment:
Should we log it as error? If I understand correctly, it's expected that latency requests will be rejected.

Reply:
No, we expect our request to turn off MRAM suspension to be accepted

Reply:
Oh, okay, I get it now. Thanks